src/Symfony/Component/Translation/DependencyInjection/TranslatorPass.php

```diff
@@ -26,6 +26,10 @@ class TranslatorPass implements CompilerPassInterface
 
     public function __construct(string $translatorServiceId = 'translator.default', string $readerServiceId = 'translation.reader', string $loaderTag = 'translation.loader', string $debugCommandServiceId = 'console.command.translation_debug', string $updateCommandServiceId = 'console.command.translation_update')
```

Comment:
We should remove any occurrences in the code base to get rid of the current deprecation notices. 

There are some in the tests code base for TranslatorPass
https://github.com/symfony/symfony/search?q=%22new+TranslatorPass%22

Reply:
Yep, see  #40470